security/bug-bounty.md

```diff
@@ -1,32 +1,61 @@
+* Issues that can't be obviously exploited, like self-XSS (an XSS that requires the user to enter a payload that is executed against themselves).
+* Login/logout CSRF.
+* Vulnerabilities affecting users of unsupported or unpatched browsers and platforms.
+
+Clever also reserves the right to deny eligibility for a vulnerability that was previously reported or discovered internally.
 * Be the first to report a specific vulnerability.
-* Send a clear textual description of the report along with steps to reproduce the vulnerability. Include attachments such as screenshots or proof of concept code as necessary.
-* Disclose the vulnerability report directly and exclusively to us. Public disclosure or disclosure to other third parties -- including vulnerability brokers -- before we addressed your report will forfeit the reward.
+* Disclose the vulnerability report directly and exclusively to us. Public disclosure or disclosure to other third parties, including vulnerability brokers, before we address your report will cause you to forfeit the reward.
 
-In general, the following would not meet the threshold for severity:
-* Vulnerabilities on sites hosted by third-parties unless they lead to a vulnerability on the main website
-* Denial of service
-* Spamming
-* Email server configuration (SPF, DKIM, and DMARC)
-* Name server configuration (DNSSec)
-* Self-XSS (an XSS that requires the user to enter a payload)
+Scope
+-----
+The following web services are in scope for this program:
+* https://clever.com (main website)
+* https://schools.clever.com (dashbaord for school districts)
+* https://apps.clever.com (dashboard for applications)
+* https://clever.com/oauth (instant login)
+* https://clever.com/in/ (portal)
+* https://requests.clever.com (districts request access to an app)
 
-Thank you for helping keep Clever safe!
+**Signing up for a school district account**
 
-**Thanks**
+1. Go to the [schools signup](https://schools.clever.com/signup) page.
+2. Complete signup, and use a new district name that starts with "#DEMO". Click "Add".
+3. Click "Step 1: Set up accounts"
+4. Select "Custom username and password" for the accounts
+5. For usernames and password, select "Clever Passwords based on SIS Data (Recommended)"
+6. For student information system, select "Other" and use any name.
+7. Upload sample data through the [Web Upload](https://schools.clever.com/settings/system/upload) interface, which you can find under Settings => Sync. You can find a set of sample .csv files there.
 
-We believe in recognizing the work of others.
+**Signing up for an application account**
 
-If your work helps us improve the security of our service. we'd be happy to [acknowledge your contribution](/Clever/thanks) and ensure you are fairly rewarded for your discovery.
+1. Go to the [application developer signup](https://apps.clever.com/signup) page.
+2. Complete signup, and use an application name that starts with "#DEMO".
```

Comment:
As a soft enforcement mechanism, can we note that failure to prepend the app name with `#DEMO` could result in forfeit of the bounty?